MekHQ/src/mekhq/gui/adapter/PersonnelTableMouseAdapter.java

```diff
@@ -2632,15 +2679,16 @@ private JMenuItem newMenuItem(String text, String command, boolean enabled) {
     }
 
     private JCheckBoxMenuItem newCheckboxMenu(String text, String command, boolean selected) {
-        return newCheckboxMenu(text, command, selected, true);
-    }
-
-    private JCheckBoxMenuItem newCheckboxMenu(String text, String command, boolean selected, boolean enabled) {
         JCheckBoxMenuItem result = new JCheckBoxMenuItem(text);
         result.setSelected(selected);
         result.setActionCommand(command);
         result.addActionListener(this);
         result.setEnabled(true);
         return result;
     }
+
+    private boolean IsNullOrEmpty(String str){
+        return str.trim().length() == 0;
+//        return ((str != null) && (!str.trim().isEmpty()));
```

Comment:
We have a method like this in megamek.common.util, let's use that instead of reinventing the wheel.

Reply:
Ah, I forgot to hit enter to ask if there was pre-existing way to do this in MekHQ. And otherwise, I would have suggested making one, cause the fact it isn't standard in Java is bloody annoying.